.eslintrc.yml

```diff
@@ -41,6 +41,7 @@
       - before: true
         after: true
   globals:
+    Promise: true
```

Comment:
i think the thing you need to do here is change ecmaVersion to 7 or 8. we shouldn't have to specify this explicitly

Reply:
7 and 8 both still say `'Promise' is not defined` and i am way too unfamiliar with npm/eslint to know what else to do :(

Reply:
add 'es6: true' to `env`